src/4_initialization.js

```diff
@@ -35,7 +35,7 @@ else if (typeof exports === 'object') {
 if (!TITANIUM_BUILD) {
 	// Always make a global.
 	if (window) {
-		window['branch'] = branch_instance;
+		window.branch = branch_instance;
```

Comment:
Our `.jscsrc` and `.jshintrc` indeed need updates. Good point! Subsequential changes appending...